software/messaging/src/main/java/brooklyn/entity/messaging/kafka/KafkaZooKeeperSshDriver.java

```diff
@@ -60,4 +67,12 @@ public Integer getZookeeperPort() {
         return getEntity().getAttribute(KafkaZooKeeper.ZOOKEEPER_PORT);
     }
 
+    @Override
+    public void createTopic(String topic) {
+        String zookeeperUrl = getEntity().getAttribute(Attributes.HOSTNAME) + ":" + getZookeeperPort();
+        newScript(CUSTOMIZING)
+                .failOnNonZeroResultCode()
+                .body.append(String.format("./bin/%s  --create --zookeeper %s --replication-factor 1 --partitions 1 --topic %s", getTopicsScriptName(), zookeeperUrl, topic))
```

Comment:
`getTopicsScriptName()` is coming from the user, right? Should be bash-escaped, see `BashStringEscapes`. Otherwise will let the user execute arbitrary commands (or fail when special characters are used).